pkg/presentation/viewmodels/models.go

```diff
@@ -116,3 +116,9 @@ type Upload struct {
 	CreatedAt string
 	UpdatedAt string
 }
+
+type Tab struct {
+	ID       string
+	Href     string
+	Position uint
```

Comment:
This should be a string, unless there is a good reason

Reply:
Don't need this in viewmodel, removed it